idemp-rest/WEB-INF/src/org/bandahealth/idempiere/rest/service/db/ChargeDBService.java

```diff
@@ -72,6 +72,11 @@ public Charge saveNonPatientPayment(Charge entity) {
 	@Override
 	public Charge saveEntity(Charge entity) {
 		// Save to the DB
+		if (entity == null) {
```

Comment:
What causes the entity to be null?

Reply:
If the payload (whether from a postman request or whatever the source) is null, it throws an error specifically at line 80 when it tries to retrieve the uuid. I encountered this issue during testing. However, if we can be sure that no request will be null, we can remove the check. Thanks

Reply:
I'm actually okay with someone sending a `null` - it should error out. If sent wrongly from postman, I don't think we need to adjust logic for that. If sent wrongly from the UI, I'd like to figure out the cause and correct it there.

Reply:
cool